Ginger/GingerCore/ALM/RQMCore.cs

```diff
@@ -140,6 +141,8 @@ public override ObservableList<Activity> GingerActivitiesRepo
             AlmConfig.AlmType = almType;
             AlmConfig.ALMConfigPackageFolderPath = ALMConfigPackageFolderPath;
             AlmConfig.JiraTestingALM = testingALMType;
+
+            AlmCoreConfig = AlmConfig;
```

Comment:
Codacy found an issue: [Remove this useless assignment to local variable 'AlmCoreConfig'.](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=6407666)